store-implementation/hbase-store/src/test/java/uk/gov/gchq/gaffer/hbasestore/NonToBytesSerialiser.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.hbasestore;
+
+import uk.gov.gchq.gaffer.commonutil.CommonConstants;
+import uk.gov.gchq.gaffer.exception.SerialisationException;
+import uk.gov.gchq.gaffer.serialisation.Serialiser;
+import java.io.UnsupportedEncodingException;
+
+public class NonToBytesSerialiser implements Serialiser<Integer, byte[]> {
```

Comment:
I don't really understand what the point of this class is? Can you just use one of the other Serialisers we already have?